HW_1/task_4.py

```diff
@@ -0,0 +1,10 @@
+''' Задача 8: Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+*Пример: *
+3 2 4 -> yes
+3 2 1 -> no
+'''
+n,m,k = int(input()),int(input()),int(input())
+if k < n * m and ((k % n == 0) or (k % m == 0)):
```

Comment:
```suggestion
if k < n * m and (k % n == 0 or k % m == 0):
```
Первое условие можно расширить по нижней границе допустимых значений переменной `k`.